website/content/en/status/report-2023-01-2023-03/clusteradm.adoc

```diff
@@ -13,7 +13,7 @@ In this quarter, the team has worked on the following:
 * Regular disk and parts support (and replacement) for all physical hosts and mirrors.
 * Improve the PowerPC package builders.
 ** With new parts obtained through the FreeBSD Foundation, the builders now have new NVMEs with heatsinks and more memory.
-It helped seize the heat issue, and they are building packages faster now.
```

Comment:
According to <https://ell.stackexchange.com/a/244800/31430>, the verb _seize_ is an old French legal term that means _to be put in legal possession of_. 

In modern English, the verb commonly used in phrases such as [seize stolen goods](https://www.startpage.com/do/dsearch?query=seize+stolen+goods&cat=web&language=english) … I would not use it in relation to a hardware issue.

Reply:
Agreed.